samples/openapi3/client/petstore/python-experimental/petstore_api/models/shape_or_null.py

```diff
@@ -74,7 +74,7 @@ class ShapeOrNull(ModelComposed):
     validations = {
     }
 
-    additional_properties_type = None
+    additional_properties_type = (bool, date, datetime, dict, float, int, list, str, none_type,)  # noqa: E501
 
     @cached_property
     def openapi_types():
```

Comment:
Note to self: the generated openapi_types() is wrong, it should include none_type.